# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1318256625 14400
#      Mon Oct 10 10:23:45 2011 -0400
# Branch stable
# Node ID b0e70a71647b671ebcfa7a79af1ae6d3c0f52065
# Parent  46f5e41c861076d6faa999be09b2b7dbb2fcbd32
Version 3.4.3 released.

* configure.ac (AC_INIT): Version number is now 3.4.3.
(OCTAVE_RELEASE_DATE): Update to 2011-10-10.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -22,23 +22,23 @@ dnl Copyright (C) 1993-2011 John W. Eato
 
 ### Preserve CFLAGS and CXXFLAGS from the environment before doing
 ### anything else because we don't know which macros might call
 ### AC_PROG_CC or AC_PROG_CXX.
 
 EXTERN_CFLAGS="$CFLAGS"
 EXTERN_CXXFLAGS="$CXXFLAGS"
 
-AC_INIT([GNU Octave], [3.4.3-rc0], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [3.4.3], [http://octave.org/bugs.html], [octave])
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 OCTAVE_API_VERSION_NUMBER="45"
 OCTAVE_API_VERSION="api-v$OCTAVE_API_VERSION_NUMBER+"
-OCTAVE_RELEASE_DATE="2011-09-29"
+OCTAVE_RELEASE_DATE="2011-10-10"
 OCTAVE_COPYRIGHT="Copyright (C) 2011 John W. Eaton and others."
 AC_SUBST(OCTAVE_VERSION)
 AC_SUBST(OCTAVE_API_VERSION_NUMBER)
 AC_SUBST(OCTAVE_API_VERSION)
 AC_SUBST(OCTAVE_RELEASE_DATE)
 AC_SUBST(OCTAVE_COPYRIGHT)
 
 AC_REVISION($Revision: 1.603 $)
